### ISSUE TITLE
iOS7 style segment control UI?

### ISSUE THREAD

**Issue by hemangshah** [samvermette/SVSegmentedControl#69]
Is it possible to create exactly same (replicate) iOS7 styled UISegementControl with SVSegmentedControl?
- I'll need to change background and text colors for selected and unselected state.
- I'll need to set color for seperator.
- I'll need to give rounded border for segement control.

Here's the sample segment like iOS7

![screen shot 2013-12-11 at 11 31 27 am](https://f.cloud.github.com/assets/3313412/1721655/2af3ad5e-622a-11e3-83bf-1596cbac148b.png)

Is it possible?

NOTE: I can get it default in iOS7 but I want the same look in iOS5 and iOS6 as well.

**Comment by marcoscurvello** [comment]
Hey @hemangshah  how did you achieve the rounded border on the segmented control ? thanks =)

**Comment by hemangshah** [comment]
@MarcosCurvello Hi.. buddy, I've asked this question to the @samvermette but I don't get any reply from him. Since I created my own for this. I make the exact sample by taking a rounded image and add it as the background of buttons. For the above case, I need 6 images in total.
1. left selected.
2. left unselected.
3. right selected.
4. right unselected.
5. center selected.
6. center unselected.

<b>then follow these steps:</b>

1 set each unselected image for normal and selected image for selected state for button
2 then set an action on each button, when a button tapped I changed 
3 make other button setSelected to NO and for the tapped button setSelected to YES
4 that's it.
5 in my need I don't want re tap once the button was tap, so I also set its userInteractions to NO and for all the other it should be YES.

I know it's not dynamic way but for an instance its much easy and headache free as I'd no other way to get that work. My app is in the store now! =)

Hope this would help you !!
